src/components/social.js

```diff
@@ -9,29 +9,57 @@ module.exports = React.createClass({
     var locale = this.context.intl.locale;
     var twitterShareURL = 'https://twitter.com/share?url=' + appURL +'/' + locale + '/&text=' + encodeURIComponent(this.context.intl.formatMessage({id: 'i_donated_to_mozilla'}));
     var facebookShareURL = 'https://www.facebook.com/sharer/sharer.php?u=' + appURL + '/' + locale + '/';
+    var emailSubject = '';
+    var emailBody = '';
```

Comment:
What's the ETA for these?

Also, these are going need to go into a properties.json file likely, as it currently stands. cc  @TheoChevalier